Source/ACE.Entity/Character.cs

```diff
@@ -169,6 +169,14 @@ public void GrantXp(ulong amount)
         }
 
         /// <summary>
+        /// adds only avaiable xp when user has already gained max level
+        /// </summary>
+        public void GrantAdditionalXp(ulong amount)
```

Comment:
This function is used to add additional experience points to an account when a user has _**already**_ reached the maximum level. The logic was added so that characters can continue to gain experience, after reaching the maximum level; in conjunction with the GrantXP(ulong amount) function, which now will requires a experience limit of 191,226,310,247, to lock a character to the max level.